tests/test_datasets/test_dataset.py

```diff
@@ -56,6 +56,92 @@ def df_and_regressors() -> Tuple[pd.DataFrame, pd.DataFrame, List[str]]:
     return df, df_exog, ["regressor_1", "regressor_2"]
 
 
+@pytest.fixture()
```

Comment:
I thought we are using `@pytest.fixture` if there are not parameters instead of `@pytest.fixture()`.

Reply:
Fixed